documentation/sdn/kolla_ansible_encryption_state.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Kolla Ansible Encryption state
+tags: kolla kolla-ansible openstack security
+---
+
+# Kolla Ansible control plane security
+
+For Kolla Ansible SCS lists 18 supported services (APIs) as part of a compliant
+Open Stack deployment. There are also a couple of support tools deployed like
+database (MariaDB), cache service (Redis), message broker (RabbitMQ), virtual
+networking components (Open vSwitch) and virtualization manager for compute
+(libvirt). They communicate mostly through REST APIs on internal virtual
+networks. This control plane traffic is of security concern. Most of the
+services and tools support TLS encryption and there have been some efforts to
+make this support accessible to operators of Kolla Ansible deployments. We
+analyzed the state of encryption in current version of Kolla Ansible (2024.1
+Series) and prepared upstream contributions to complete the control plane
+traffic encryption coverage.
+
+## Current state of control plane encryption
+
+![](https://input.scs.community/uploads/7f141b31-d23c-498f-ae0b-ac74127c7ce4.png)
+[This table](https://scs.sovereignit.de/nextcloud/s/G4DzznMKeJHqMBx)
+shows current state of the encryption support, vertically we have
+services and tools used and horizontally communication channels they are using.
+Green cells show encryption already supported in Kolla Ansible, yellow ones are
+the new contributions to the project currently on review. Grey cells are just
+combinations which do not communicate in real deployment. There is also a
+more
+[detailed table version](https://scs.sovereignit.de/nextcloud/s/G4DzznMKeJHqMBx)
+containing also the contribution links.
+
+Kolla Ansible currently divides the communication channels into 3 categories -
+internal, external and backend. Internal and external are both API client to
+proxy communications, between services on internal networks and external
+clients to proxy respectively. Settings for encrypting these path are mostly
```

Comment:
Consider reformulating the sentence: `Internal and external .... `

Suggestion: 

Internal and external communications involve clients connecting to the proxy: internal communications occur between services on internal networks, while external communications connect external clients to the proxy.